### PR TITLE
Add `Select::onChange()` method

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: src/Tag/Select.php [hunk 1]
```diff
@@ -90,6 +90,19 @@ public function items(Optgroup|Option ...$items): self
         return $new;
     }
 
+    /**
+     * @see https://html.spec.whatwg.org/multipage/webappapis.html#handler-onchange
+     * @see https://html.spec.whatwg.org/multipage/indices.html#event-change
+     *
+     * @param string $value The script to be run when select value is changed.
+     */
+    public function onChange(string $value): self
```
Comment (samdark) [inline src/Tag/Select.php:99]: I don't think we should add it:

1. It's not a good practice to use such attributes with inline JavaScript.
2. There are tons of such events for every tag. Are we going to add every possible event?

**Review by samdark — CHANGES_REQUESTED** [review]

File: src/Tag/Select.php (hunk 1 above)
Comment (vjik) [reply src/Tag/Select.php:99]: 1. I agree, this HTML attribute is rare used. However, I have needed it a few times, and make it with this method more convient than `attribute()` method usage.

2. Step by step, when they are needed in practice.

Comment (samdark) [reply src/Tag/Select.php:99]: I don't think it is a good idea:

image

Comment (vjik) [reply src/Tag/Select.php:99]: 👌

**Pull request closed without merging**